asciidoctorj-api/src/main/java/org/asciidoctor/Options.java

```diff
@@ -79,7 +91,7 @@ public void setAttributes(Map<String, Object> attributes) {
      *
      */
     public void setHeaderFooter(boolean headerFooter) {
-        this.options.put(HEADER_FOOTER, headerFooter);
+        this.options.put(STANDALONE, headerFooter);
```

Comment:
Is it ok to change this already?
There might for example be postprocessors relying on this option being present when the client code calls .setHeaderFooter().

Reply:
You are right, I restored it, thanks!